components/Input/index.js

```diff
@@ -8,6 +8,7 @@ const Input = ({ placeholder, onPressEnter }) => {
 
   return (
     <InputStyled
+      role="input"
```

Comment:
Woops, my fault! 
```suggestion
```

Reply:
Actually, it must have the `role="input"` because I test the `input` render using this. Disconsider this comment :)